src/mina-signer/src/sign-zkapp-command.unit-test.ts

```diff
@@ -81,28 +81,32 @@ expect(stringify(dummyInput.packed)).toEqual(
   stringify(dummyInputSnarky.packed)
 );
 
-test(Random.accountUpdate, (accountUpdate) => {
-  fixVerificationKey(accountUpdate);
+test(
+  Random.accountUpdate,
+  RandomTransaction.networkId,
```

Comment:
nice!

Reply:
Not really, it probably needs to be aligned and not just random.

Reply:
Or perhaps your previous [point](https://github.com/o1-labs/o1js/pull/1437#discussion_r1491355883) needs to be addressed first...

Reply:
Or if `Provable.inCheckedComputation()` is always true then `accountUpdateSnarky.hash()` will always fallback to the `testnet`.

Reply:
Yes you need to address the other case

Reply:
Updated it like [this](https://github.com/o1-labs/o1js/pull/1437/commits/1984c25ca58075b6124dbe8c26e3bbe324001fbb). 
@mitschabaude is that true that in order to make it pass we first should address your previous [point](https://github.com/o1-labs/o1js/pull/1437#discussion_r1491355883) above?

Reply:
that looks good, and yes you need to address my previous point